content/_includes/Jobs.md

```diff
@@ -0,0 +1,18 @@
+---
+---
+
+The **Jobs** <span class="material-icons">assignment</span> icon button displays all running and failed jobs/processes. Minimized jobs/processes can be accessed here.
+
+![JobsMenu](/images/SCALE/JobsMenu.png "TrueNAS SCALE Task Manager")
+
+Click on a running task to display the dialog window for that running task.
```

Comment:
we moved away from using window in reference to a dialog so suggest the following
```suggestion
Click on a running task to display a dialog for that running task.
```